.github/workflows/release-npm.yml

```diff
@@ -15,11 +15,9 @@ on:
       tag:
         description: 'Npm dist-tag'
         default: 'latest'
+        required: false
```

Comment:
Logic change here as well.

Reply:
no logic change, as default value for `required` is false if not set, see github docs